specs/2019-04/Windows-Plan.md

```diff
@@ -0,0 +1,98 @@
+# App Center Windows Application Proposal 
+
+This plan details the proposed improvements for App Center’s Windows application offering along with a comparison of App Center and HockeyApp. This is work in progress and will be iterated upon based on any internal updates and feedback from the community. We expect to start work on the planned items in the second quarter of 2019. 
+
+
+## App Center’s Current Offering 
+### Diagnostics
+
+- Crash reporting for UWP (.NET native) apps distributed through the Windows store 
+> Note: there are some known limitations and missing features for UWP apps. Find more details in our [Windows documentation](https://docs.microsoft.com/en-us/appcenter/diagnostics/windows-support). 
+
+### Distribution
+
+- Distributing UWP and Unity UWP apps using the APPX or MSIX package format
+- Unofficial support to distribute WPF and WinForms apps when packaged as APPX or MSIX using the [Windows Application Packaging](https://docs.microsoft.com/en-us/windows/uwp/porting/desktop-to-uwp-packaging-dot-net)
+
+### Analytics 
+
+- Analytics SDK for UWP apps and all of its functionalities listed in App Center's [analytics documentation](https://docs.microsoft.com/en-us/appcenter/analytics/).  
+
+### Build
+- Build support for UWP apps
+
+## Planned Improvements 
+
+### Diagnostics 
+
+- Crash reporting capabilities for store and sideloaded UWP (.NET Native) and WPF (.NET Framework) apps aligned with other platforms supported in App Center. Find more details on App Center Diagnostics features in our [documentation](https://docs.microsoft.com/en-us/appcenter/diagnostics/features).  
```

Comment:
Any reason we don't also add Windows Forms? I would imagine the App Center infrastructure for WPF and WinForms crash analytics is identical (and there are tons of WinForms developers).

Reply:
Hi @diverdan92 - I spoke to our engineering teams and it doesn't seem like adding support for WinForms will require too much additional work. We will commit to supporting WinForms and I've updated the plan to reflect this. Thanks for the feedback!